CTFd/utils/__init__.py

```diff
@@ -28,20 +22,12 @@ def _get_config(key):
         if value and value.isdigit():
             return int(value)
         elif value and isinstance(value, six.string_types):
-            if value.lower() == 'true':
-                return True
-            elif value.lower() == 'false':
-                return False
-            else:
-                return value
+            return {'true': True, 'false', False}.get(value.lower(), value)
 
 
 def get_config(key, default=None):
     value = _get_config(key)
-    if value is None:
-        return default
-    else:
-        return value
+    return default if value is None else value
```

Comment:
This reads okay but I am not entirely sure what the benefit is here besides slightly more condensed code. If there's no benefit besides that please revert.

Reply:
Would __return value or default__ work in this situation?